### PR TITLE
Add support for percentage tiers at plan and subscription le…

### PR DESCRIPTION
**Pull request by camilopineda100**

### CODE REVIEW

**Review by seckenrode — COMMENTED** [review]
This looks good from my side, but I'm going to tag @btruncali1 in for a light review since he's got more professional C# experience

File: Library/CurrencyPercentageTier.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    public List<PercentageTier> PercentageTiers
+    {
+        get { return _percentageTiers ?? (_percentageTiers = new List<PercentageTier>()); }
+        set { _percentageTiers = value; }
+    }
+
+    #region Constructors
+
+    public CurrencyPercentageTier() {
+    }
```
Comment (btruncali1) [inline Library/CurrencyPercentageTier.cs]: id do an empty constructor in 1 line, but up to you totally

File: Library/CurrencyPercentageTier.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    internal CurrencyPercentageTier(XmlTextReader xmlReader){
+      ReadXml(xmlReader);
```
Comment (btruncali1) [inline Library/CurrencyPercentageTier.cs:28]: ifd do this but up to you

`internal CurrencyPercentageTier(XmlTextReader xmlReader)
{

      ReadXml(xmlReader);

 }`

Comment (btruncali1) [reply Library/CurrencyPercentageTier.cs:28]: yikes its hard to get styling right in these comments

**Review by btruncali1 — APPROVED** [review]

**Pull request merged**